src/Avalonia.Headless/AvaloniaHeadlessPlatform.cs

```diff
@@ -51,6 +51,11 @@ class HeadlessWindowingPlatform : IWindowingPlatform
             public IWindowImpl CreateEmbeddableWindow() => throw new PlatformNotSupportedException();
 
             public IPopupImpl CreatePopup() => new HeadlessWindowImpl(true);
+
+            public ITrayIconImpl CreateTrayIcon()
+            {
+                throw new NotImplementedException();
```

Comment:
It that exception handled by the xplat code?

Reply:
nope... whats preferred throw exception or return null?

Reply:
handles returning null.